host/sgx/sgxquoteprovider.c

```diff
@@ -50,290 +53,238 @@ oe_result_t oe_initialize_quote_provider()
     return result;
 }
 
-oe_result_t oe_get_revocation_info(oe_get_revocation_info_args_t* args)
+oe_result_t oe_get_sgx_quote_verification_collateral(
```

Comment:
Would it be possible to just return and use the sgx_ql_qve_collateral_t returned from the DCAP client instead of copying the data into our oe_get_sgx_quote_verification_collateral_args_t structure? This would shave off some time.

Reply:
Thanks for the suggestion, will open an issue and address them later.